dashboard-app/src/util/dataManipulation/aggregatedToTableData.ts

```diff
@@ -18,7 +18,14 @@ export const isDateString = (x: any): boolean => {
 };
 
 export const abbreviateDateString = (x: string): string => {
-  return moment(new Date(x)).format(Months.format.table);
+  const [month, year] = x.split(' ');
```

Comment:
Is this all necessary because the `MMM YY` string isn't parsed by the date constructor in Firefox?

Reply:
At this point date is represented by `MMMM YYYY` eg January 2013. Moment only handles dates that are iso or rfc2822 format so it is passed to the date constructor instead (functionality that is going to be removed in future releases of `moment`).

This could be avoided if we use different month string formatting in the aggregation layer which we can look into as part of #101